### ISSUE TITLE
Tool Tips

### ISSUE THREAD

**Issue by hitechpanchal** [tjackenpacken/taskbar-groups#295]
Is it not possible to have tooltips within the group? I cannot get any tips. Although a small group is manageable, with a a big group, i think this would be very helpful.

**Comment by staadexpert** [comment]
I recently started using this tool to group all IDE icons. I have multiple versions of Visual Studios with similar-looking icons, and I keep clicking on the wrong icon. A tooltip would be handy in such situations.

**Comment by pdawcewicz** [comment]
Yes, I have the same problem. I was wandering why there is a possibility to change apps' names withing group definiction, and these names are not used as tooltips.
